.github/workflows/ci.yml

```diff
@@ -2,8 +2,12 @@ name: CI
 on:
   push:
     branches: [main]
+    paths-ignore:
+    - '!docs/**'
   pull_request:
     branches: [main]
+    paths-ignore:
+    - '!docs/**'
```

Comment:
Shouldn't this be just `'docs/**'` without the `!`? Otherwise, I think it wouldn't have any effects.

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-excluding-paths

Reply:
This is a mistake.